Styles.css

```diff
@@ -478,7 +672,7 @@ hr.new1 {
 .contact {
   display: flex;
   flex-direction: column;
-  background-image: url(./images/contact-bg.png);
+  background-image: url(images/contact-bg.png);
   background-color: #fff;
   background-repeat: no-repeat;
   background-size: 100% 100%;
```

Comment:
- [ ] Impressive work you've done here. However, you'll notice there are a few adjustments needed on placing some of the background images on the contact section of your desktop version. The best way to see this is to visualize the difference, of which I've made a detailed comparison below- This change is needed to help you stick to design, and user requirements just as you would for clients in the future 👍 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/230055397-6b194707-803e-402f-a9b8-0acdcd3d283e.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/230055499-33affd33-39b0-460e-8c31-f023a33fcb2e.png)